contrib/camel/config/207-camel-k-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: camel-k
+    camel.apache.org/component: operator
+  name: camel-k-operator
+  namespace: knative-sources
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      camel.apache.org/component: operator
+      name: camel-k-operator
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        camel.apache.org/component: operator
+        name: camel-k-operator
+    spec:
+      containers:
+      - command:
+        - camel-k
+        env:
+        - name: WATCH_NAMESPACE
+          value: ""
+        - name: OPERATOR_NAME
+          value: camel-k
+        image: docker.io/apache/camel-k:0.2.0
+        imagePullPolicy: IfNotPresent
+        name: camel-k-operator
+        ports:
+        - containerPort: 60000
```

Comment:
looks like a non-standard metrics port, where does this come from?

Reply:
Nice catch! It's not used anymore: https://github.com/operator-framework/operator-sdk/issues/1130